mantle/cmd/kola/testiso.go

```diff
@@ -92,14 +92,17 @@ var (
 	}
 	tests_s390x = []string{
 		"iso-live-login.s390fw",
+		"iso-live-login.4k.s390fw",
 		"iso-offline-install.s390fw",
+		"iso-offline-install.4k.s390fw",
 		// https://github.com/coreos/fedora-coreos-tracker/issues/1434
 		// "iso-offline-install.mpath.s390fw",
-		// https://github.com/coreos/fedora-coreos-tracker/issues/1261
-		// "iso-offline-install.4k.s390fw",
 		"pxe-online-install.s390fw",
+		"pxe-online-install.4k.s390fw",
 		"pxe-offline-install.s390fw",
+		"pxe-offline-install.4k.s390fw",
 		"miniso-install.s390fw",
+		"miniso-install.4k.s390fw",
 		"miniso-install.nm.s390fw",
 		"miniso-install.4k.nm.s390fw",
```

Comment:
```suggestion
		"miniso-install.nm.s390fw",
		"miniso-install.4k.nm.s390fw",
```

We probably don't need a `miniso-install.4k.s390fw` and `miniso-install.4k.nm.s390fw` test.

Reply:
why?

Reply:
Each test takes up time and resources.

If you notice here these lists of tests don't have all the possible options that we could combine. What we tried to do was throw enough combinations together to get good enough coverage without having to run a huge matrix of tests. I'm making the argument here that we don't need to add a second `miniso-install` tests that touches 4k disks because we proably already have enough coverage with the `miniso-install.4k.nm.s390fw` test.

Reply:
Got it, will drop 'em